packages/nextra/src/client/components/tabs/tabs.tsx

```diff
@@ -101,9 +101,9 @@ export function Tabs({
           <HeadlessTab
             key={index}
             disabled={isTabObjectItem(item) && item.disabled}
-            className={({ selected, disabled, hover, focus }) =>
+            className={({ selected, disabled, hover }) =>
               cn(
-                focus && 'nextra-focusable _ring-inset',
+                'nextra-focus focus:_ring-inset',
```

Comment:
```suggestion
focus && 'nextra-focusable _ring-inset',
selected && '_outline-none',
```

the fix is simpler we should hide outline for `selected` state